packages/@react-spectrum/breadcrumbs/src/Breadcrumbs.tsx

```diff
@@ -25,6 +25,47 @@ import {useProviderProps} from '@react-spectrum/provider';
 const MIN_VISIBLE_ITEMS = 1;
 const MAX_VISIBLE_ITEMS = 4;
 
+function hasResizeObserver() {
+  return typeof window.ResizeObserver !== 'undefined';
+}
+
+type useResizeObserverOptionsType<T> = {
+  ref: RefObject<T>,
+  onResize: () => void
+}
+
+function useResizeObserver<T extends HTMLElement>(options: useResizeObserverOptionsType<T>) {
+  const {ref, onResize} = options;
+
+  useEffect(() => {
+    if (!ref) { return; }
+
+    if (!hasResizeObserver()) {
+      window.addEventListener('resize', onResize, false);
+      return;
```

Comment:
this return will prevent cleanup of the addEventListener down on line 57
you can either return the cleanup function here
or you can remove the return and use an else to avoid running the ResizeObserver code